### DISCUSSION TITLE
Image available from userdata for css backgrounds

### DISCUSSION THREAD

**Discussion by baumrasen** [photobooth-app/photobooth-app#302] [Ideas]
I would like to show a preview of the collage as background for the buttons. 
![image](https://github.com/photobooth-app/photobooth-app/assets/7682804/1363fb0f-d28e-4bd1-80e1-b82c8e0f6968)

The left button uses an image from the gallery. That works fine.
But the right button uses an image from the userdata-folder. I think, it wood be better, to store the images there.

from the private.css:
```
 .action-button-1 {
    background-color: green !important;
    background-image: url("/userdata/frames/3pics-wood-default.png") !important;
    padding: 50px 100px !important;
    background-size: 300px 200px !important;
    background-repeat: no-repeat !important;
    background-clip: border-box !important;
    border: 10px solid green !important;
 }
```

I'm not good with CSS. So probably the CSS code is only a dirty hack. I don't like the fix padding values.

**Comment by mgineer85** [comment]
The css probably is fine.
Currently the userdata folder is used only by internal processes with the exception private.css that is served by the Webserver also. The other files are not part of the Webserver.

pls send this issue as feature request, I will check for a simple implementation.
Until then just use the data folder for testing. The files could be deleted if you click the delete gallery button in admin dashboard, so better keep a copy
